### PR TITLE
Feat(Query): Log Retention Is Not Greater Than 90 Days for Alicloud Terraform

### PR DESCRIPTION
**Pull request by cxAndreFelicidade**
**Proposed Changes**
- Query: Log Retention Is Not Greater Than 90 Days for Alicloud Terraform

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/terraform/alicloud/log_retention_is_not_greater_than_90_days/metadata.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "id": "ed6cf6ff-9a1f-491c-9f88-e03c0807f390",
+  "queryName": "Log Retention Is Not Greater Than 90 Days",
+  "severity": "MEDIUM",
+  "category": "Observability",
+  "descriptionText": "OSS Bucket should have logging enabled for longer than 90 days, for better visibility of resources and objects.",
```
Comment (rafaela-soares) [inline assets/queries/terraform/alicloud/log_retention_is_not_greater_than_90_days/metadata.json:6]:
```suggestion
  "descriptionText": "OSS Log Store should have logging enabled for longer than 90 days, for better visibility of resources and objects.",
```

File: assets/queries/terraform/alicloud/log_retention_is_not_greater_than_90_days/query.rego [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    rperiod := resource.retention_period
+    to_number(rperiod) < 90
+    
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("alicloud_log_store[%s].retention_period", [name]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": sprintf("For a log storing resource to exist and have retetion period set to 90+ days.", [name]),
+		"keyActualValue": sprintf("The retention period for the resource %s is not over 90 days", [name]),
+        "searchLine": common_lib.build_search_line(["resource", "alicloud_disk", name, "encrypted"], []),
```
Comment (rafaela-soares) [inline assets/queries/terraform/alicloud/log_retention_is_not_greater_than_90_days/query.rego:33]:
```suggestion
        "searchLine": common_lib.build_search_line(["resource", "alicloud_log_store", name, "encrypted"], []),
```

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

**Comment by fjsnogueira** [comment]
Logo
**Checkmarx AST – Scan Summary & Details** – [5e036028\-58b6\-4dda\-8428\-77fc0f920e90](https://eu.ast.checkmarx.net/projects/0244d5f3-72f2-4c91-8775-654a746fd66f/scans?id=5e036028-58b6-4dda-8428-77fc0f920e90&branch=log_retention_is_not_greater_than_90_days)
### No New Or Fixed Issues Found

**Review by rafaela-soares — APPROVED** [review]
LGTM 🚀

File: assets/queries/terraform/alicloud/ros_stack_without_template/metadata.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  "queryName": "Ros Stack Without Template",
+  "severity": "MEDIUM",
+  "category": "Build Process",
+  "descriptionText": "Alicloud ROS Stack should have a template defined through the attribute template_url or attribute template_body",
+  "descriptionUrl": "https://registry.terraform.io/providers/aliyun/alicloud/latest/docs/resources/ros_stack",
+  "platform": "Terraform",
+  "descriptionID": "29378f62",
+  "cloudProvider": "aws"
+}
+
```
Comment (rafaela-soares) [inline assets/queries/terraform/alicloud/ros_stack_without_template/metadata.json:12]: Please, delete this query since PR #5262 covers it.

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

**Review by rafaela-soares — APPROVED** [review]

**Review by joaorufi — APPROVED** [review]

**Pull request merged**